travel_platform/travel_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-05-27 08:05
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Query',
```

Comment:
This file was created automatically during manage.py migrate command.
I know that there should be properly model, but have no idea, what should it be.